python/ray/test/test_modin.py

```diff
@@ -10,3 +10,7 @@ def test_modin_import():
     frame_data = [1, 2, 3, 4, 5, 6, 7, 8]
     frame = pd.DataFrame(frame_data)
     assert frame.sum().squeeze() == sum(frame_data)
+
+
+if __name__ == '__main__':
+    test_modin_import()
```

Comment:
Actually, this `main` should be removed. Pytest tests don't use `__main__`.

Reply:
This is not run with pytest, that's the reason why the test wasn't run.

Reply:
done